pkg/transfer/transfer.go

```diff
@@ -68,6 +69,11 @@ type ImageGetter interface {
 	Get(context.Context, images.Store) (images.Image, error)
 }
 
+// ImageExporter exports images to a writer
```

Comment:
Would it more accurate to say "... to a content store"?

Reply:
Actually it exports images to a local file stream, like a local tarball, I am not sure if it is a content store as well.